### ISSUE TITLE
Instance log

### ISSUE THREAD

**Issue by grepwood** [killpowa/Twickt-Launcher#1094]
Would be nice to know what the hell the instance is printing out when trying to run it as it can sometimes crash without offering me a log. Most of the time this happens when I deplete open file handles on the host.

**Comment by embeddedt** [comment]
This button opens the log for me:

![image](https://user-images.githubusercontent.com/42941056/135770527-47d87d2d-3ebb-4e1f-9bee-84d4ebae5a97.png)

**Comment by grepwood** [comment]
Thank you! <3

**Comment by grepwood** [comment]
Sorry, but that's completely useless. That just brings up Chrome dev console, not the Java log of the started instance.
![x](https://user-images.githubusercontent.com/4225739/136676705-2028513d-d1d7-41fa-878e-113cce97aab7.png)

**Comment by huantianad** [comment]
@grepwood go into the console tab and launch an instance

**Comment by grepwood** [comment]
This is also incredibly uninformative.
![obraz](https://user-images.githubusercontent.com/4225739/136676724-e93a97e5-1470-4b3d-a9ec-ccc2e3a73f15.png)
Might as well be this:
![obraz](https://user-images.githubusercontent.com/4225739/136676761-1dfbb660-56e9-4831-9f97-eba53a19c35c.png)

**Comment by huantianad** [comment]
> This is also incredibly uninformative.
> ![obraz](https://user-images.githubusercontent.com/4225739/136676724-e93a97e5-1470-4b3d-a9ec-ccc2e3a73f15.png)
> Might as well be this:
> ![obraz](https://user-images.githubusercontent.com/4225739/136676761-1dfbb660-56e9-4831-9f97-eba53a19c35c.png)

I do agree it is uninformative, but right now it just displays the process error. There’s no good way to show exactly what caused the crash other than showing the entire log

**Comment by grepwood** [comment]
@huantianad thank you, it is indeed in the console tab.
Sorry for venting. This is by far the most potent launcher much to where credit is due, but also one that drives way different than anything else. I could leave this thread open if this area needs improvement for user experience parity between other launchers.

**Comment by huantianad** [comment]
Yeah it definitely is something that can be improved on, not sure if there's an existing issue on this yet.
